drivers/at/at.c

```diff
@@ -210,6 +269,7 @@ ssize_t at_send_cmd_get_resp(at_dev_t *dev, const char *command,
 {
     ssize_t res;
 
+    xtimer_usleep(AT_CMD_DELAY);
```

Comment:
`xtimer` is added but not as a dependency to the driver (you might consider using `ztimer` as well)